dosagelib/plugins/x.py

```diff
@@ -7,17 +7,17 @@
 from ..helpers import bounceStarter
 
 
-class Xkcd(_ParserScraper):
+class XKCD(_ParserScraper):
```

Comment:
Why?

Reply:
I actually had a reason for this one, if a bit pedantic. From https://xkcd.com/about/:
> **How do I write "xkcd"? There's nothing in Strunk and White about this.**
For those of us pedantic enough to want a rule, here it is: The preferred form is "xkcd", all lower-case. In formal contexts where a lowercase word shouldn't start a sentence, "XKCD" is an okay alternative. "Xkcd" is frowned upon.

Reply:
Between a rock and a hard place: Python code style leans a bit to `Xkcd` as a "proper" class name, I think...

Reply:
I took a quick look through the existing comics; it looks like several already have capitalized class names (DMFA, PS238, SMBC, SSDD, etc.).